pages/index.js

```diff
@@ -1,67 +1,145 @@
-            className={styles.card}
-          >
-            <h3>Deploy &rarr;</h3>
+  return (
+    <Link href="/sign-up">
+      <a>
+        <div className={styles.card}>
+          <img src="/icons/user-plus.svg" />
+          <div>
+            <h3>Sign up for an account</h3>
             <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
+              Sign up and sign in to explore all the features provided by Clerk
+              out-of-the-box
             </p>
-          </a>
+          </div>
+          <div className={styles.arrow}>
+            <img src="/icons/arrow-right.svg" />
+          </div>
         </div>
-      </main>
+      </a>
+    </Link>
+  );
+});
+
+// Main component
+const Main = () => (
+  <main className={styles.main}>
+    <h1 className={styles.title}>Welcome to your new app</h1>
+    <p className={styles.description}>Sign up for an account to get started</p>
 
-      <footer className={styles.footer}>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{" "}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
+    <div className={styles.cards}>
+      <ConditionalComponent />
+
+      <Link href="https://dashboard.clerk.dev">
+        <a>
+          <div className={styles.card}>
+            <img src="/icons/settings.svg" />
+            <div>
+              <h3>Configure settings for your app</h3>
+              <p>
+                Visit Clerk to manage instances and configure settings for user
+                management, theme, and more
+              </p>
+            </div>
+            <div className={styles.arrow}>
+              <img src="/icons/arrow-right.svg" />
+            </div>
+          </div>
+        </a>
+      </Link>
+    </div>
+
+    <div className={styles.links}>
+      <Link href="https://docs.clerk.dev">
+        <a className={styles.link}>
+          <span className={styles.linkText}>Read Clerk documentation</span>
         </a>
-      </footer>
+      </Link>
+      <Link href="https://nextjs.org/docs">
+        <a className={styles.link}>
+          <span className={styles.linkText}>Read NextJS documentation</span>
+        </a>
+      </Link>
+    </div>
+  </main>
+);
+
+// Footer component
+const Footer = () => (
+  <footer className={styles.footer}>
+    <a
+      href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Powered by{" "}
+      <img src="/clerk.svg" alt="Clerk Logo" className={styles.logo} />
+      +
+      <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
```

Comment:
I think it makes more sense to put the nextjs logo here?

Reply:
Done